kcbq-connector/src/main/java/com/wepay/kafka/connect/bigquery/GcpClientBuilder.java

```diff
@@ -45,15 +48,18 @@ public enum KeySource {
   }
 
   private static final Logger logger = LoggerFactory.getLogger(GcpClientBuilder.class);
-
+  private static final String USER_AGENT_HEADER_KEY = "user-agent";
+  private static final String USER_AGENT_HEADER_FORMAT = "%s (GPN: Confluent;) Google BigQuery Sink/%s";
+  private static String userAgent;
```

Comment:
should this be a static variable? In the future we may have multi version plugins so we should avoid this.

Reply:
@kpatelatwork I didn't understand the multi version plugin part here. Although I will make this change to make it non static.

Reply:
Changed.